webapp/src/routes/index.js

```diff
@@ -1,6 +1,8 @@
 import NotFound from './NotFound'
 import InternalError from './InternalError'
 import Profile from './Profile'
+
```

Comment:
remove empty line

Reply:
I think you removed something that is needed.

Take a look at this compiling error:
<img width="491" alt="Screen Shot 2021-06-14 at 08 37 05" src="https://user-images.githubusercontent.com/28828796/121910442-15802580-ccec-11eb-9765-0194f5aafe7b.png">

Reply:
Don't delete it, the merge only brought that without the import